### PR TITLE
PostProcHandlerTest: Don't skip test in testPurgePageOnQueryDependency

### PR DESCRIPTION
**Pull request by paladox**
## Summary by CodeRabbit

- **Bug Fixes**
	- Removed an inactive test case to streamline the testing suite.

- **Tests**
	- Enhanced test methods to validate the behavior of the `PostProcHandler` class under various conditions using mock objects.
	- Updated mock creation process for improved efficiency.

### CODE REVIEW

**Comment by paladox** [comment]
@coderabbitai review

**Comment by paladox** [comment]
@coderabbitai review

File: tests/phpunit/PostProcHandlerTest.php [hunk 1]
```diff
@@ -260,15 +260,17 @@ public function testPurgePageOnQueryDependency() {
 			]
 		);
 
-		$this->markTestSkipped(
-			"Check smwLikelyOutdatedDependencies set up for MW >= 1.39"
+		$title = $this->createMock( '\Title' );
+
+		$dependencyLinksValidator = $this->createMock( '\SMW\SQLStore\QueryDependency\DependencyLinksValidator' );
+		$namespaceExaminer = $this->createMock( '\SMW\NamespaceExaminer' );
+		$entityCache = $this->createMock( '\SMW\EntityCache' );
```
Comment (JeroenDeDauw) [inline tests/phpunit/PostProcHandlerTest.php:267]: Remember you can use `YourName::class`

Comment (paladox) [reply tests/phpunit/PostProcHandlerTest.php]: Thanks, done.

**Comment by paladox** [comment]
@coderabbitai review

**Pull request merged**